empress/recon_vis/test_render.py

```diff
@@ -0,0 +1,22 @@
+import empress.recon_vis.recon_viewer
```

Comment:
We use this to test our code without having to run the entire GUI. Can we keep this in here for as long as possible :)

Reply:
Yes, you could keep the code as is, but please move the code to `draw/draw_recon_viewer.py` and create a function called `test_render_3` in a similar way other drawing tests are organized (using `fig.save` instead of `fig.show`).

Reply:
fixed